test/FESpacesTests/FEAutodiffTests.jl

```diff
@@ -162,4 +165,65 @@ gridapgradj = assemble_vector(gradient(j_Λ,uh),U)
 fdgradj = ForwardDiff.gradient(j_Λ_,θ)
 test_array(gridapgradj,fdgradj,≈)
 
+# testing jacobians of functionals involving Skeleton inetgration terms
+uh = FEFunction(U,rand(num_free_dofs(U)))
+θ = get_free_dof_values(uh)
+dv = get_fe_basis(V)
+du = get_trial_fe_basis(U)
+
+a(uh,vh) = ∫(mean(uh)*mean(vh))*dΛ
+b(uh,vh) = ∫(-jump(uh*n_Λ)⊙mean(∇(vh)) +
+              jump(vh*n_Λ)⊙mean(∇(uh)) +
+              mean(Δ(uh))*mean(Δ(vh))  +
+              jump(uh*n_Λ)⊙jump(vh*n_Λ))*dΛ
+g(uh,vh) = ∫(mean(uh*vh))*dΛ
+h(uh,vh) = ∫(jump(uh*vh*n_Λ)⊙jump(vh*vh*n_Λ))*dΛ
+j(uh,vh) = ∫(mean(∇(vh)*uh)⊙jump((∇(vh)⊙∇(uh))*n_Λ))*dΛ
+
+function _change_input(f,θ,uh)
+  dir = similar(uh.dirichlet_values,eltype(θ))
+  U = uh.fe_space
+  uh = FEFunction(U,θ,dir)
+  T = eltype(θ)
```

Comment:
Can we add here a comment on why we needed to use the strongly-typed lower-level interfaces of `SparseMatrixAssembler`?